plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/commit/CommitViewImpl.java

```diff
@@ -180,7 +181,12 @@ public void setEnableCommitButton(boolean enable) {
     /** {@inheritDoc} */
     @Override
     public void focusInMessageField() {
-        message.setFocus(true);
+        new Timer() {
```

Comment:
Try to do this with scheduler like https://github.com/codenvy/che-core/blob/88d75e053098e2455fa926df33916f88afdcb6c3/ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/annotation/AnnotationModelImpl.java#L169-L175
To avoid unclear delay 300